imports/api/groceries.js

```diff
@@ -38,6 +40,14 @@ Meteor.methods({
 
 		return Groceries.remove(groceryId);
 	},
+	'groceries.archive'(groceryId, archived = true) {
```

Comment:
What does the `archived = true` do?

Reply:
It sets the default value of `archived` to `true`